embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -67,6 +67,10 @@
         @ConfigDefault("null")
         public Optional<String> getQuery();
 
+        @Config("use_raw_query")
+        @ConfigDefault("false")
+        public boolean getUseRawQuery();
```

Comment:
This property will be used only for incremental loading.
Users will understand easily if the property name shows that (for example, it contains `incremental`).

Reply:
How about `use_query_with_incremental`?
Or do you have any favorite candidates?

Reply:
Thank you for the proposition.
How about `use_raw_query_with_incremental` ?
`query` will be used even if the property is `false`.
(But the property name is a bit longer...)

Reply:
Sounds good. I'm going to replace with `use_raw_query_with_incremental`